app/controllers/items/partial_undesignate.js

```diff
@@ -13,57 +15,88 @@ export default Ember.Controller.extend({
   partialUndesignateBackLinkpath: Ember.computed.localStorage(),
 
   init() {
-    this.set('promptUserPopUp', false);
+    this.set("promptUserPopUp", false);
+  },
+
+  cannotDispatchItems(designation) {
+    return (
+      designation &&
```

Comment:
can we set this up in a const and return that const?? As it would be more readable

Reply:
yeah, that can be done.